lighthouserc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  ci: {
+    collect: {
+      url: ['http://localhost:3000/'],
```

Comment:
This will only hit for `localhost`, is that right? Say if this was to run on a production will this still work?

Reply:
Yes, it will be running inside a ubuntu-latest container.

Reply:
Cool!